app/gui/src/project-view/components/shared/AgGridTableView.vue

```diff
@@ -144,13 +145,22 @@ function onGridReady(event: GridReadyEvent<TData>) {
   gridApi.value = event.api
 }
 
+watch(
+  () => props.reloadGrid,
+  () => {
+    if (rowModelType.value != 'clientSide') gridApi.value?.refreshServerSide({ purge: true })
+  },
```

Comment:
Such cases are handled by exposing a method in component - see https://vuejs.org/api/sfc-script-setup#defineexpose and example usage: https://github.com/enso-org/enso/blob/develop/app/gui/src/project-view/components/widgets/AutoSizedInput.vue#L95